landing-page/types/alces-branding/content/styles/branding.css

```diff
@@ -18,9 +18,17 @@ body {
 .navbar .dropdown-menu {
     background-color: rgba(0, 0, 0, 0.9);
 }
```

Comment:
I think we need a corresponding rule for `.navbar .dropdown-item` to set the colour to white.  That should fix the webapp issue dropdown item colour issue.